apps/server/src/api-data/automation/clients/ontime.client.ts

```diff
@@ -15,7 +16,8 @@ export function toOntimeAction(action: OntimeAction) {
     case 'aux1-pause':
       return auxTimerService.pause(1);
     case 'aux1-set': {
-      return auxTimerService.setTime(action.time, 1);
+      const time = parseUserTime(action.time);
```

Comment:
@cpvalente 
not sure at what point we should make it a number
what do you thing?